src/components/partials/help-guide-lb-business.js

```diff
@@ -0,0 +1,105 @@
+import { html } from 'lit-html';
```

Comment:
what does this file do? should the data not be inside form.io and a shim is used to make it pretty?

Reply:
Partials are pulled in by the help guide component to show that part, developed separately and imported in where needed.